### ISSUE TITLE
[Feature] Allow configuration of sensitive values with a path to a file

### ISSUE THREAD

**Issue by ethnt** [0xERR0R/blocky#694]
There are a few places where there could be sensitive values in the Blocky configuration, namely in the database configuration options. A nice-to-have would be the option to use a path to a file instead of the actual value. For example:

`/run/secrets/redis_password`
```
letmein
```

`/etc/blocky/config.yml`
```yaml
redis:
  passwordFile: /run/secrets/redis_password
```

**Comment by kwitsch** [comment]
I'm currently developing #375.
This would merge file configurations with environment variable configuration.

In other projects of mine I load docker secrets with the exact name of environment variables as such.

Example: /run/secrets/BLOCKY_REDIS_PASSWORD -> content:"password value" would be the handled as environment variable BLOCKY_REDIS_PASSWORD

Would this suffice your needs?

**Comment by ThinkChaos** [comment]
Environment variables are less secure since they are inherited by processes by default, and in the case of Docker it's even worse: they're global.  
A file allows to fine tune who/what can access the content using permissions and ACLs.

This is kind of achievable at the moment by relying on our support for multiple config files in a directory: split `redis: password:` into its own file with more restrictive permissions.  
Obviously this is not as user friendly as having a `passwordFile` option, so I think it'd be worth investigating what we can do to make this split easier once #375 lands.

**Comment by marcosrosse** [comment]
Hi 👋🏿 ,

Is this also related with the database password exposed in the log? I configured a PostgreSQL for the QuerLog, but  when I saw the container log the user and password was exposed there.
